test/accelerators/NNPA/numerical/CMakeLists.txt

```diff
@@ -51,31 +63,42 @@ llvm_update_compile_flags(rapidcheck)
 # but it did not compile when I removed these two. TODO, figure out why.
 set(TEST_LINK_LIBS rapidcheck ModelLib)
 
-# TODO: Support tests for Conv
-# add_numerical_unittest(TestConvNNPA
-#   ${ONNX_NUMERICALTEST_SRC_DIR}/TestConv.cpp
-#   LINK_LIBS PRIVATE ${TEST_LINK_LIBS}
-#   )
+# Conv2D
+set(TestConvNNPA_instruction zdnn_conv2d)
+set(TestConvNNPA_data_range 0.1)
```

Comment:
Could you please explain general settings of `data_range` and `instruction` in `docs/AccelNNPAHowToUseAndTest.md`? Is it possible to use an upperbound and lowerbound for a range instead of a single value, which is more flexible?

Reply:
Updated docs.